docs/1.0. EDID to Receiver Caps mapping.md

```diff
@@ -65,6 +65,8 @@ The timing descriptors MAY include one or more of the following mappings:
 - `urn:x-nmos:cap:format:interlace_mode`
 - `urn:x-nmos:cap:meta:preference`
 
+Each of these Parameter Constraints MUST use `enum` Constraint Keyword with a single item in the array.
```

Comment:
I am not sure this makes sense for devices with scalers 🤔

Reply:
If "scaling" here means changing video resolution before output (Full HD came from the network, Receiver sends 4K to Sink), then I don't think this change effects scalers. Receiver should just list what it supports on its input which can be done with `enum`.